docs/user-guide/dpo.rst

```diff
@@ -2,17 +2,23 @@
 
 .. _model-aligner-dpo:
 
-Model Alignment by Direct Preference Optimization (DPO)
-@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+Model Alignment by DPO, RPO and IPO
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
 
 The NeMo Framework supports efficient model alignment via the NeMo-Aligner codebase.
 
 All algorithms in NeMo-Aligner will work with any GPT-based model that is from Megatron Core (in the config it has ``mcore_gpt=True``). For the purposes of this tutorial, we will go through the entire DPO pipeline using the newly released `2B GPT model with 4096 sequence length <https://huggingface.co/nvidia/GPT-2B-001>`__.  The same tutorial also works for GPT models (such as LLaMa2) of any size.
```

Comment:
define first use of acronym DPO as follows.

All algorithms in NeMo-Aligner will work with any GPT-based model that is from Megatron Core (in the config it has ``mcore_gpt=True``). For the purposes of this tutorial, we will go through the entire Direct Preference Optimization (DPO) pipeline using the newly released `2B GPT model with 4096 sequence length <https://huggingface.co/nvidia/GPT-2B-001>`__.  The same tutorial also works for GPT models (such as LLaMa2) of any size.

Reply:
```suggestion
All algorithms in NeMo-Aligner will work with any GPT-based model that is from Megatron Core (in the config it has mcore_gpt=True). For the purposes of this tutorial, we will go through the entire Direct Preference Optimization (DPO) pipeline using the newly released 2B GPT model with 4096 sequence length <https://huggingface.co/nvidia/GPT-2B-001>__. The same tutorial also works for GPT models (such as LLaMa2) of any size.
```